kong/cmd/utils/prefix_handler.lua

```diff
@@ -378,6 +378,49 @@ local function write_process_secrets_file(path, data)
   return true
 end
 
+local function pre_create_lmdb(conf)
+  local prefix = conf.prefix
+  local lmdb_path = conf.lmdb_environment_path or "dbless.lmdb"
+  local user = string.match(conf.nginx_user or "", "%w+") or "$USER"
+
+  local dir_name
+  if lmdb_path:sub(1, 1) == "/" then
+    dir_name = lmdb_path
+  else
+    dir_name = prefix .. "/" .. lmdb_path
+  end
+
+  if pl_path.isdir(dir_name) then
+    return true
+  end
+
```

Comment:
Add a log line in here to aid debugging. Like:

"LMDB directory " .. dir_name .. " does not exist, pre-creating with the correct permissions"

Reply:
done